engine/src/main/java/org/destinationsol/ModuleManager.java

```diff
@@ -27,17 +28,16 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.terasology.assets.ResourceUrn;
-import org.terasology.module.Module;
-import org.terasology.module.ModuleEnvironment;
-import org.terasology.module.ModuleFactory;
-import org.terasology.module.ModulePathScanner;
-import org.terasology.module.ModuleRegistry;
-import org.terasology.module.TableModuleRegistry;
+import org.terasology.module.*;
```

Comment:
For a couple of reasons, we don't like to use * imports, even if we do use all classes in the package. IntelliJ probably did this by default, so please revert back to the individual imports.

Reply:
Please refer to https://stackoverflow.com/questions/3587071/disable-intellij-starred-package-imports for tips on how to deal with this in the future.